async-graphql-demo/src/main.rs

```diff
@@ -0,0 +1,38 @@
+mod starwars;
+
+use actix_web::{guard, web, App, HttpResponse, HttpServer, Result};
+use async_graphql::http::{playground_source, GraphQLPlaygroundConfig};
+use async_graphql::{EmptyMutation, EmptySubscription, Schema};
+use async_graphql_actix_web::{GQLRequest, GQLResponse};
+use starwars::{QueryRoot, StarWars, StarWarsSchema};
+
+async fn index(schema: web::Data<StarWarsSchema>, req: GQLRequest) -> GQLResponse {
+    schema.execute(req.into_inner()).await.into()
+}
+
+async fn index_playground() -> Result<HttpResponse> {
+    Ok(HttpResponse::Ok()
+        .content_type("text/html; charset=utf-8")
+        .body(playground_source(
+            GraphQLPlaygroundConfig::new("/").subscription_endpoint("/"),
+        )))
+}
+
+#[actix_rt::main]
```

Comment:
This is the recommended way now and can remove the actix-rt dependency.

```suggestion
#[actix_web::main]
```

Reply:
Thank you! 🙂